Update for tiddlyweb 1.3.0

cdent opened the pull request:
Set up to use httpexceptor

I note that the tests aren't passing for me, this appears to be a
slight different in how the diff output is presented, probably the
result of a version difference in difflib.

Sorry this is on my master, figured as it's a one off...

merge and release please and thank you?

cdent committed:
Update for tiddlyweb 1.3.0

which means set up to use httpexceptor

I note that the tests aren't passing for me, this appears to be a
slight different in how the diff output is presented, probably the
result of a version difference in difflib.

FND commented:
Tests are failing for me as well - I'm currently not in a position to dig deeper, so this might have to wait, sorry.

```
E    assert '--- \n\n+++ ...rem foo ipsum' == '---  \n\n+++ ...rem foo ipsum'
E      - --- 
E      + ---  
E      ?     +
E        
E      - +++ 
E      + +++  
E      ?     +
E        
E      - @@ -1 +1 @@
E      + @@ -1,1 +1,1 @@
E      ?      ++   ++
E        
E        -lorem ipsum
E        +lorem foo ipsum
```

cdent committed:
Make expected be more what is expected.

Apparently there have been some whitespace changes at the ends of lines
and in how a single line is indicated.

cdent commented:
That last commit gets the test passing with a diff that is correct:

" In many versions of GNU diff, each range can omit the comma and trailing value s, in which case s defaults to 1"

and the whitespace part is apparently the result of different attitudes towards missing context info

so this is should be okay now

FND commented:
Merged and released - thanks!
